src/components/Button/Button.plugin.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const _ = require('lodash')
+const chroma = require('chroma-js')
+const defaultConfig = require('tailwindcss/defaultConfig')()
```

Comment:
pull in tailwind.js here so that we are using custom values from our theme, this pulls in the defaults from the tailwind module

Reply:
I think I had issues here with cyclical dependencies. Will try it again.